main.nf

```diff
@@ -939,6 +912,62 @@ process fetch_MTseqs {
 
 }
 
+process sentieon_mitochondrial_qc {
+
+    // Fetch mitochondrial coverage statistics
+    // Calculate mean_coverage and pct_above_500x
+    
+    cpus 52
+    memory '30 GB'
+	tag "$id"
+	cache 'deep'
```

Comment:
I doubt this is needed. It sometimes strains i/o. It's a remnant from an older version of nextflow where it refused to resume the original sentieon qc-process

Reply:
Removed from both sentieon_qc and _mitochondrial_qc